src/components/com_tc/administrator/views/usertcs/view.html.php

```diff
@@ -61,7 +61,7 @@ public function display($tpl = null)
 		$comParams = ComponentHelper::getParams('com_tc');
 		$this->dateFormat = $comParams->get('date_format_show');
 
-		if ($dateFormat == "custom")
+		if ($this->dateFormat == "custom")
```

Comment:
@paritshivani If you are using dateFormat variable only in this file then don't make it global. Use $dateFormat

Reply:
we are using it in default.php also